man/rgbgfx.1

```diff
@@ -164,6 +165,37 @@ for a list of formats and their descriptions.
 .It Fl d Ar depth , Fl \-depth Ar depth
 Set the bit depth of the output tile data, in bits per pixel (bpp), either 1 or 2 (the default).
 This changes how tile data is output, and the maximum number of colors per palette (2 and 4 respectively).
+.It Fl i Ar input_tiles , Fl \-input-tileset Ar input_tiles
+Read tiles that will be used to convert this image, and that will always be given the first IDs.
+.Ar input_tiles
+must contain tile data in
+.Dq raw
+format, as generated through the
+.Fl o
+option.
+.Pp
+If used together with
+.Fl o ,
+then the image can contain tiles not in
+.Ar input_tiles .
+Otherwise, the image must be able to be generated using
+.Em only
+the tiles from
+.Ar input_tiles ,
+and thus generate different tile data.
+The former is more useful if you want several images to share a given set of tiles, such as different levels sharing a single tileset; the latter, if you want to control more precisely the numeric IDs of specific tiles.
+.Pp
+If more than one color palette is in use, it is also
+.Sy strongly
+advised to dump it together with the tile data, and to pass it using
+.Fl c Cm gbc: Ns Ar input_palette .
+This is because
+.Nm
+may not pack the palettes the same way that it did when generating
+.Ar input_tiles .
```

Comment:
```suggestion
If the image will use more than one color palette, it is
.Em strongly
advised to generate the palette set along with the input tile data, and pass
.Fl c Cm gbc: Ns Ar input_palette
along with
.Fl i input_tiles .
This is because
.Nm
may not generate an identical palette set for this image as for its input tileset.
```

Reply:
Not sure this wording is good, but I think the same of mine anyway, so let's use yours.